tests/algorithms/test_required_on_load.py

```diff
@@ -21,6 +26,10 @@ def initialize_algorithm(algo_cls: Type):
     elif algo_cls == algorithms.GatedLinearUnits:
         pytest.importorskip('transformers')
         return algo_cls()
+    elif algo_cls == algorithms.Factorize:
+        return algo_cls(min_features=48, latent_features=24)
+    elif algo_cls == algorithms.SqueezeExcite:
+        return algo_cls(min_channels=32)
```

Comment:
If someone changes the fixture, test will start failing since we test to verify it transforms the right amount of layers. Since this initialization is a little more general and used for the other test as well, I'm fine not being super picky here. It's not ideal, but idt theres a risk of silent stopping so I'm ok with it.